pubspec.yaml

```diff
@@ -1,24 +1,27 @@
 name: vector_map_tiles
 description: A plugin for `flutter_map` that enables the use of vector tiles.
-version: 4.0.0
+version: 5.0.0
 homepage: https://github.com/greensopinion/flutter-vector-map-tiles
 
 environment:
-  sdk: ">=2.18.0 <4.0.0"
-  flutter: ">=2.14.0"
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=3.10.0"
```

Comment:
are these version constraints required by `flutter_map`?

Reply:
Yep, we now require these minimums.